BridgeAppSDK/SBASubtaskStep.swift

```diff
@@ -55,7 +55,7 @@ public class SBASubtaskStep: ORKStep {
     
     func substepIdentifier(identifier: String) -> String? {
         guard let range = identifier.rangeOfString("\(self.subtask.identifier).") else { return nil }
-        let stepRange = Range(start: range.endIndex, end: identifier.endIndex)
+        let stepRange = range.endIndex ..< identifier.endIndex
```

Comment:
actually seeing it in this context, that would fit--start just past the end of the matched substring, and go to the end of the whole identifier string by stopping just before its end marker.